bulbs/special_coverage/static/special_coverage/stylesheets/bulbs_sc.css

```diff
@@ -0,0 +1,290 @@
+.image {
```

Comment:
At present, yes. We do not have a SASS compiler in the ClickHole's pipeline, and both the Onion and CH use LESS for their main stylesheets. I'll add this css to the pipeline of both sites for now.